fraud/sync.go

```diff
@@ -0,0 +1,148 @@
+package fraud
+
+import (
+	"context"
+	"time"
+
+	"github.com/ipfs/go-datastore"
+	"github.com/libp2p/go-libp2p-core/event"
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/libp2p/go-libp2p-core/peer"
+	pubsub "github.com/libp2p/go-libp2p-pubsub"
+
+	"github.com/celestiaorg/go-libp2p-messenger/serde"
+
+	pb "github.com/celestiaorg/celestia-node/fraud/pb"
+)
+
+// syncFraudProofs encompasses the behavior for fetching fraud proofs from other peers.
+// syncFraudProofs subscribes to EvtPeerIdentificationCompleted to get newly connected peers
+// to request fraud proofs from and request fraud proofs from them.
+// After fraud proofs are received, they are published to all local subscriptions for verification
+// order to be verified.
+func (f *ProofService) syncFraudProofs(ctx context.Context) {
+	log.Debug("start fetching Fraud Proofs")
```

Comment:
I think maybe we should pick a casing and stick with it, I've seen both "Fraud Proofs" and "fraud proofs" in many places and it looks awkward sometimes

Reply:
let's do it lower case